Dockerfile

```diff
@@ -877,6 +887,14 @@ ENV SYFT_FORMAT_SPDX_JSON_CREATED_TIME=${SOURCE_DATE_EPOCH}
 ARG NAME
 ARG TAG
 
+COPY --from=pkg-containerd-amd64 /usr/bin/containerd-shim-runc-v2 /tmp
```

Comment:
I would add some comment here for the future readers on what's going on here :)

Reply:
Ah, yeah, worth it!

Reply:
Oh, great, yes, I guess I should do a unit test then, if we've got rootfs there. Thanks for linking to buildinfo, will implement